index.js

```diff
@@ -64,6 +64,12 @@ const {argv} = require('yargs')
         default: true,
         type: 'boolean',
     })
+    .option('concurrency', {
+        alias: 'u',
```

Comment:
Is `u` a good shorthand for `concurrency`. Isn't it more common to use the two first letters when a single character is already in use for something else?

Reply:
Will change to batch-size and `s` :-)